codex-rs/tui/src/history_cell.rs

```diff
@@ -242,34 +246,39 @@ impl HistoryCell {
             exit_code,
             stdout,
             stderr,
-            duration,
+            duration: _,
         } = output;
 
         let mut lines: Vec<Line<'static>> = Vec::new();
-
-        // Title depends on whether we have output yet.
-        let title_line = Line::from(vec![
-            "command".magenta(),
-            format!(
-                " (code: {}, duration: {})",
-                exit_code,
-                format_duration(duration)
-            )
-            .dim(),
-        ]);
-        lines.push(title_line);
+        let command_escaped = strip_bash_lc_and_escape(&command);
+        lines.push(Line::from(vec![
+            "⚡Ran command ".magenta(),
```

Comment:
```suggestion
            "⚡ Ran command ".magenta(),
```
space after the bolt? love the emoji tho